test/test/mocks/digital-exchange/components.js

```diff
@@ -1,5 +1,5 @@
-// LIST_DE_PLUGINS_OK - List Digital Exchange plugins.
-export const LIST_DE_PLUGINS_OK = [
+// LIST_DE_COMPONENTS_OK - List Digital Exchange components.
```

Comment:
let's avoid unnecessary comments. if a comment is necessary to explain the name of the variable it means that the name needs improving.